frontend/src/Css/login.css

```diff
@@ -3,133 +3,124 @@
-    text-align: center;
+#screenshot {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 600px;
+  -webkit-box-shadow: 5px 5px 15px 5px rgb(0 0 0 / 55%);
+  box-shadow: 5px 5px 15px 5px rgb(0 0 0 / 55%);
 }
 
-#headline{
-    font-size: 36px;
-    margin-bottom: 20px;
+#right-column {
+  display: grid;
+  align-items: center;
+  justify-content: center;
 }
 
-#subheadline{
-    font-size: 22px;
+#form-wrapper {
+  width: 80%;
 }
 
-#screenshot{
-    margin-top: 20px;
-    margin-bottom: 20px;
-    width: 600px;
-    -webkit-box-shadow: 5px 5px 15px 5px rgb(0 0 0 / 55%);
-    box-shadow: 5px 5px 15px 5px rgb(0 0 0 / 55%);
+#title {
+  font-size: 72px;
+  padding-left: 40px;
 }
 
-#right-column{
-    display: grid;
-    align-items: center;
-    justify-content: center;
+.un {
+  width: 76%;
+  color: rgb(38, 50, 56);
+  font-weight: 700;
+  font-size: 14px;
+  letter-spacing: 1px;
+  background: rgba(136, 126, 126, 0.04);
+  padding: 10px 20px;
+  border: none;
+  border-radius: 20px;
+  outline: none;
+  box-sizing: border-box;
+  border: 2px solid rgba(0, 0, 0, 0.02);
+  margin-bottom: 50px;
+  text-align: center;
+  margin-bottom: 27px;
+  font-family: 'Ubuntu', sans-serif;
 }
 
+form.form1 {
+  padding-top: 40px;
+}
 
-#form-wrapper{
-    width: 80%;
+.pass {
+  width: 76%;
+  color: rgb(38, 50, 56);
+  font-weight: 700;
+  font-size: 14px;
+  letter-spacing: 1px;
+  background: rgba(136, 126, 126, 0.04);
+  padding: 10px 20px;
+  border: none;
+  border-radius: 20px;
+  outline: none;
+  box-sizing: border-box;
+  border: 2px solid rgba(0, 0, 0, 0.02);
+  margin-bottom: 50px;
+  text-align: center;
+  margin-bottom: 27px;
+  font-family: 'Ubuntu', sans-serif;
 }
 
-#title{
-    font-size: 72px;
-    padding-left: 40px;
+.un:focus,
+.pass:focus {
+  border: 2px solid rgba(0, 0, 0, 0.18) !important;
 }
 
-.un {
-    width: 76%;
-    color: rgb(38, 50, 56);
-    font-weight: 700;
-    font-size: 14px;
-    letter-spacing: 1px;
-    background: rgba(136, 126, 126, 0.04);
-    padding: 10px 20px;
-    border: none;
-    border-radius: 20px;
-    outline: none;
-    box-sizing: border-box;
-    border: 2px solid rgba(0, 0, 0, 0.02);
-    margin-bottom: 50px;
-    text-align: center;
-    margin-bottom: 27px;
-    font-family: 'Ubuntu', sans-serif;
-    }
-    
-    form.form1 {
-        padding-top: 40px;
-    }
-
-
-    .pass {
-        width: 76%;
-        color: rgb(38, 50, 56);
-        font-weight: 700;
-        font-size: 14px;
-        letter-spacing: 1px;
-        background: rgba(136, 126, 126, 0.04);
-        padding: 10px 20px;
-        border: none;
-        border-radius: 20px;
-        outline: none;
-        box-sizing: border-box;
-        border: 2px solid rgba(0, 0, 0, 0.02);
-        margin-bottom: 50px;
-        text-align: center;
-        margin-bottom: 27px;
-        font-family: 'Ubuntu', sans-serif;
-        }
-
-
-.un:focus, .pass:focus {
-    border: 2px solid rgba(0, 0, 0, 0.18) !important;
-    
+.submit {
```

Comment:
Was `.un` removed?
(Maybe the edits to the rest of the stuff could have been added in a second commit?)

Reply:
Nothing has been removed. Just prettier did it's job.